src/frontend-samples/viewer-only-2d-sample/ViewerOnly2dUI.tsx

```diff
@@ -34,13 +50,36 @@ export default class ViewportOnly2dUI extends React.Component<{}, ViewportOnly2d
   private _getSampleUi = (iModelName: SampleIModels) => {
     return new SampleWidgetUiProvider(
       "The picker below shows a list of 2D models in this iModel.",
-      <ControlsWidget />,
+      <ControlsWidget twoDState={this.state.twoDState} />,
       { iModelName, onIModelChange: this._changeIModel }
     );
   };
 
+  private _getDrawingModelList = (models: ModelProps[]) => {
```

Comment:
I wouldn't have functions for _getDrawingModelList and _getSheetModelList.  Rather this code would be directly in the widget.  My reason is because all this building of options is just a UI detail, and not an important part of the sample.